j1/api/api_client.py

```diff
@@ -0,0 +1,96 @@
+
+import requests
```

Comment:
More of a general thing, but what's the state of async/await in the python ecosystem? I checked the requests library and it linked out to this: https://github.com/encode/httpx (which I apparently have starred but never used). Wondering if it's worth using or if it makes things a bigger headache?